### ISSUE TITLE
I want browsers to automatically fix contrast issues

### ISSUE THREAD

**Issue by aarongustafson** [WebWeWant/webwewant.fyi#132]
Color contrast is one of the most severe accessibility issues on the web today. Browsers should be able to increase the color contrast on many pages at the request of the user. Perhaps the user could select a contrast threshold, then the browser would make an attempt to change the text color to achieve a reasonable approximation of that threshold.

<hr>

https://webwewant.fyi/wants/95/
